index.js

```diff
@@ -617,6 +617,15 @@
     function(x) { return RegExp_._test(x) && !x.global; }
   );
 
+  //# NonNegativeInteger :: Type
+  //.
+  //. Type comprising `0`, `-0`, and every [`PositiveInteger`][] value.
+  //. Also known as the set of natural numbers under ISO 80000-2:2009.
+  var NonNegativeInteger = NullaryTypeWithUrl(
+    'sanctuary-def/NonNegativeInteger',
+    function(x) { return PositiveInteger._test(x) || x === 0; }
```

Comment:
It just occurred to me that `Integer._test(x) && x >= 0` may be a clearer expression. What do you think?

We could also update the description:

_Type comprising every non-negative [`Integer`][1] value (including `-0`)._


[1]: https://github.com/sanctuary-js/sanctuary-def#Integer